shared/ff_util/include/ff_util/ff_action.h

```diff
@@ -361,13 +365,29 @@ class FreeFlyerActionClient {
     // Feedback has been received after a result before. Stop tracking the goal
     // so that this doesn't happen
     sac_->stopTrackingGoal();
+    // The response we send depends on the state
+    result_ = result;
+
     // The response we send depends on the state
     if (action_state == actionlib::SimpleClientGoalState::SUCCEEDED)
-      Complete(FreeFlyerActionState::SUCCESS, result);
+      state_response_ = FreeFlyerActionState::SUCCESS;
     else if (action_state == actionlib::SimpleClientGoalState::PREEMPTED)
-      Complete(FreeFlyerActionState::PREEMPTED, result);
+      state_response_ = FreeFlyerActionState::PREEMPTED;
     else
-      Complete(FreeFlyerActionState::ABORTED, result);
+      state_response_ = FreeFlyerActionState::ABORTED;
+
+    StartOptionalTimer(timer_response_delay_, to_response_delay_);
+  }
+
+  // Called when a result is received
+  void ResultDelayCallback(ros::TimerEvent const& event) {
+    // Feedback has been received after a result before. Stop tracking the goal
+    // so that this doesn't happen
```

Comment:
I think this comment can be removed. It doesn't match the code.

Reply:
done :)